etc/evince.profile

```diff
@@ -38,6 +38,8 @@ private-etc fonts
 
 #private-lib - seems to be breaking on Gnome Shell 3.26.2, Mutter WM, issue 1711
 #private-lib evince,libpoppler-glib.so.8
+# private-lib fix slipped through the cracks of earlier, reverted commit
```

Comment:
should this comment be in here?

Reply:
Hi @SkewedZeppelin. That comment is indeed superfluous and was more of a note-to-self kind of thing. An old habit I'll have to shake. Thanks for being thorough and raising the question here, exactly the 'spirit' I've come to appreciate as a day-to-day user of firejail.  I'll take it out asap.